src/sql/properties.sql

```diff
@@ -32,7 +38,16 @@ select
     END
 FROM sys.dm_os_sys_info
 
-	, (select top 1 service_tier = sku, virtual_core_count, hardware_generation, max_storage_gb = reserved_storage_mb/1024
-	from master.sys.server_resource_stats
-	where start_time > DATEADD(mi, -7, GETUTCDATE())
+    , (select top 1 service_tier = sku, virtual_core_count, hardware_generation, max_storage_gb = reserved_storage_mb/1024
+    from master.sys.server_resource_stats
+    where start_time > DATEADD(mi, -7, GETUTCDATE())
     order by start_time desc) as srs
+
+    , (select top 1
+    case
+        when physical_name like 'C:\ManagedDisks%' then 'True'
+        else 'False'
+    end as [Next-Gen General Purpose]
+    from sys.master_files
+    where database_id = 1
+    and type_desc = 'log') as mf
```

Comment:
In sys.server_resource_stats, we still use General Purpose for sku. The only engine telemetry at the moment that will tell us if edition is Next-Gen GP is if we parse file paths of databases and file path starts with C:\ManagedDisks